decidim/decidim
# Fix webpack version to <5.83.0

#### :tophat: What? Why?
Avoid webpack 5.83 being installed.

#### :pushpin: Related Issues
- Fixes #10877 

#### Testing
1. Build a new instance of Decidim
2. Run the server
3. See assets are compiled successfully

:hearts: Thank you!
